metricbeat/module/system/socket/socket.go

```diff
@@ -77,9 +77,10 @@ func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 	if err != nil {
 		return nil, err
 	}
-	if os.Geteuid() != 0 {
-		logp.Info("socket process info will only be available for " +
-			"metricbeat because the process is running as a non-root user")
+	if !ptable.Privileged() {
+		logp.Info("socket process info will only be available for metricbeat process " +
```

Comment:
This code is resused by Auditbeat so hard-coding "metricbeat" isn't ideal. How about

```suggestion
		logp.Info("socket process info will only be available for the Beat process " +
```

Reply:
Ok, I will change this.